pybluedot/cli.py

```diff
@@ -1,7 +1,46 @@
-"""
-cli.py constructs a cli interface of commands, options, and subcommands in argparse, given data structures from commands.py
-"""
 import argparse
-import json
-from pybluedot.commands import cmd, subcmd 
+from pybluedot.commands import cmd_arg
+from pybluedot.commands import sub_cmd
 
+#   Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
For all files, could you move this licensing statement above the imports? Should be the first thing at the top so you don't have to jump over this to see your imports.